src/components/search/search.js

```diff
@@ -25,7 +25,7 @@ class Search extends React.Component {
     ) : (
       <SearchProvider
         config={{
-          apiConnector: connector,
+          apiConnector: props.connector || connector,
```

Comment:
This change is backwards-compatible.

Reply:
One suggestion that would help keep our props organized is to add `connector` to defaultProps (down on line 103) with the value of doc's engine (lines 9-13).

Then on this like we can change to:
```suggestion
          apiConnector: props.connector,
```

defaultProps is where we define the initial values and then if the component receives a specific prop value then it will override with the value passed to that component.